### PR TITLE
tags: fix `make tags' target for out of tree builds

### PR DESCRIPTION
**Pull request by paulidale**
- [ ] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

**Review by t8m — APPROVED** [review]

File: Configurations/unix-Makefile.tmpl
Comment (tmshort) [inline Configurations/unix-Makefile.tmpl:1384]: After we go into the `$(SRCDIR)` will `util/ctags.sh` be present? (i.e. is it copied over)

Comment (t8m) [reply Configurations/unix-Makefile.tmpl:1384]: The `util/ctags.sh` is actually in $(SRCDIR) not in the build dir.

Comment (paulidale) [reply Configurations/unix-Makefile.tmpl:1384]: Yes, it's part of the source distro which is based from SRCDIR.

**Comment by paulidale** [comment]
Ping @openssl/committers for second review please.

**Review by beldmit — APPROVED** [review]
LGTM

**Comment by paulidale** [comment]
Merged, thanks for the reviews.

**Pull request closed without merging**
